### ISSUE TITLE
[P-2] Community facility place field in wrong order

### ISSUE THREAD

**Issue by ew-andrew-sus** [ytgov/yukon-ca#243]
**Description:**

_Compare:_
D7: https://yukon.ca/en/places/yukon-archives-hours-and-location
D10: https://test-yukon-drupal-10.pantheonsite.io/places/yukon-archives-hours-and-location

_Should be:_
Address
Main telephone
About / Services
Accessibility
Manager
Opening times
Holiday hours

**Task:**
Fix Community facility place field order

**Deliverable:**
Community facility place field order should be the same as on D7

**Comment by w3creatives** [comment]
Please check:

https://yukon-new-d10.w3creatives.com/docroot/en/places/yukon-archives-hours-and-location

Manager and Accessibility fields have a different order because this was configured in a way where >> Places record type >> has About/Services and Manager in the same paragraph and cannot be separated in D10.

**Comment by leeomara** [comment]
Yes, this is fine.
